src/ajax/script.js

```diff
@@ -46,6 +46,7 @@ jQuery.ajaxTransport( "script", function( s ) {
 		return {
 			send: function( _, complete ) {
 				script = jQuery( "<script>" )
+					.attr( "crossOrigin", s.crossOrigin || null )
 					.attr( s.scriptAttrs || {} )
```

Comment:
That's what the `scriptAttrs` option if for. You can add any attribute in there with no need to introduce a new option. It was designed specifically to avoid option-creep.

The correct route would be to add `crossOrigin` into the `scriptAttrs` object.

Reply:
Updated.